baybe/constraints/validation.py

```diff
@@ -98,3 +101,25 @@ def validate_cardinality_constraints_are_nonoverlapping(
                 f"cannot share the same parameters. Found the following overlapping "
                 f"parameter sets: {s1}, {s2}."
             )
+
+
+def validate_no_interpoint_and_cardinality_constraints(
```

Comment:
why is this not part of the function `validate_constraints` above? see eg line 39